### ISSUE TITLE
Can't checkout repo

### ISSUE THREAD

**Issue by IIIypuk09** [horsicq/XELFViewer#42]
Hi,
Repo can't be checkout due to folder name '?' in the images on windows. it's also not displayed in repo browser

```
Resolving deltas: 100% (15151/15151), done.
error: invalid path 'images/?/AdamHenault.json'
fatal: unable to checkout working tree

```
```
c:\git\temp\XELFViewer>git restore --source=HEAD :/
error: invalid path 'images/?/AdamHenault.json'
error: invalid path 'images/?/AdamHenault.png'
error: invalid path 'images/?/Dan0xE.json'
error: invalid path 'images/?/DavClark.json'
error: invalid path 'images/?/DavClark.png'
error: invalid path 'images/?/IntegralTech.json'
error: invalid path 'images/?/IntegralTech.png'
error: invalid path 'images/?/SpriteOvO.json'
error: invalid path 'images/?/SpriteOvO.png'
error: invalid path 'images/?/hmstk.json'
error: invalid path 'images/?/hmstk.png'
error: invalid path 'images/?/miT231-spec.json'
error: invalid path 'images/?/miT231-spec.png'
```

**Comment by horsicq** [comment]
Ok! Thanks you!

**Comment by IIIypuk09** [comment]
> Ok! Thanks you!
It's also applicable to another your repositories. Could you check and fix them too?

**Comment by horsicq** [comment]
> > Ok! Thanks you!
> 
> It's also applicable to another your repositories. Could you check and fix them too?

Sure!

I fixed it: https://github.com/horsicq/XELFViewer/commit/2e7ee4d233dbc21cb5b2fcc65dcaca66f73e1313

Could you check it again?

**Comment by IIIypuk09** [comment]
> Sure!
> 
> I fixed it: [2e7ee4d](https://github.com/horsicq/XELFViewer/commit/2e7ee4d233dbc21cb5b2fcc65dcaca66f73e1313)
> 
> Could you check it again?

Thanks! All checked out successfully including submodules.
